src/mount/fuse/mount_config.cc

```diff
@@ -90,6 +90,7 @@ struct fuse_opt gSfsOptsStage2[] = {
 	SFS_OPT("bandwidthoveruse=%lf", bandwidthoveruse, 1),
 	SFS_OPT("sfsdirentrycachesize=%u", direntrycachesize, 0),
 	SFS_OPT("nostdmountoptions", nostdmountoptions, 1),
+	SFS_OPT("DANGEROUS_sfsignoreflush", ignoreflush, 1),
```

Comment:
Consider renaming the variable to match naming conventions. If the DANGEROUS tag for this option is to be highlighted, consider doing it in the documentation (help command, man pages, etc).

Reply:
You point out to something here, which I have just checked. It seems we don't have updated our man pages, at least the readworkers and maxreadaheadrequests are not present there. I must add them also.

I'll rename it to sfsignoreflush, but as a first word in the help and man page comment I'll the word DANGEROUS. I think it is very important that people know they may lose data when using this option.